core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/order/dao/OrderDaoImpl.java

```diff
@@ -460,4 +462,30 @@ protected Long getDatabaseOrderLockTimeToLive() {
         List<Order> orders = query.getResultList();
         return orders != null ? orders : new ArrayList<Order>();
     }
+    
+    @Override
+    public List<Order> readBatchOrders(int start, int pageSize) {
```

Comment:
This method seems redundant. There's already a readBatchOrders method where you can pass in a list of qualified statuses already. That should be used instead of this one

Reply:
Removed